packages/babel-plugin-transform-async-functions/src/index.js

```diff
@@ -1,5 +1,7 @@
+import { default as syntaxAsyncFunctions } from "babel-plugin-syntax-async-functions";
```

Comment:
import syntaxAsyncFunctions from "babel-plugin-syntax-async-functions";

would be more consistent with the rest of our imports.